src/main/java/gregtech/common/metatileentities/multi/multiblockpart/appeng/MetaTileEntityMEStockingHatch.java

```diff
@@ -294,6 +295,14 @@ protected void readConfigFromTag(NBTTagCompound tag) {
         super.readConfigFromTag(tag);
     }
 
+    @Override
+    public void refreshBeforeConsumption() {
+        if (isWorkingEnabled() && autoPull) {
+            refreshList();
+            syncME();
+        }
+    }
```

Comment:
Why is this logic different from the item hatch?

Reply:
Fixed in new commit.